github-dark.css

```diff
@@ -3033,8 +3033,9 @@
   .discussion-item-header, .discussion-sidebar-heading,
   .org-header-meta .meta-item .meta-link, .underline-nav-item,
   .boxed-group-inner, .lead, .form-checkbox .note, .bot-identifier,
-  .dashboard-notice .dismiss:hover, .text-gray, td.icon .octicon-file-text,
-  td.icon .octicon-file-symlink-file, .UnderlineNav-item, .project-header-link,
+  .dashboard-notice .dismiss:hover, .text-gray,
+  td.icon .octicon-file-text, td.icon .octicon-file-symlink-file, td.icon .octicon-file-directory,
+  .UnderlineNav-item, .project-header-link,
   button.project-header-link, td.icon, .project-pane-close, .facebox-close,
   .tree-browser-result mark {
     color: #949494 !important;
```

Comment:
Please add the above change to the end of the selector list (i.e. move it next to the opening `{`) to make it easier to see the change.

Reply:
Done